airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""
+Module exposing the tests command to test airbyte-ci projects.
+"""
+
+import asyncclick as click
+from pipelines.cli.lazy_group import LazyGroup
+
+
+@click.group(
+    cls=LazyGroup,
+    help="Commands related to formatting.",
+    lazy_subcommands={
+        "java": "pipelines.airbyte_ci.format.java.commands.java",
+        "js": "pipelines.airbyte_ci.format.js.commands.js",
+        "license": "pipelines.airbyte_ci.format.license.commands.license",
+        "python": "pipelines.airbyte_ci.format.python.commands.python",
+    },
+    invoke_without_command=True,
+)
+@click.option("--fix/--check", type=bool, default=None, help="Whether to automatically fix any formatting issues detected.  [required]")
```

Comment:
So this was a weird one. If you make it required you get a more confusing error message, something like 
```
missing required property `--fix` 
```
Instead of indicating what the 2 flags are. This was my DX compromise. but looks like your suggestion [here](https://github.com/airbytehq/airbyte/pull/31831#discussion_r1373690008)  might be a better route in any case